assign2/style.css

```diff
@@ -0,0 +1,133 @@
+body {
+	margin: 0;
+	padding: 0;
+	font-family: Raleway;
+}
+
+.container {
+	font-weight: 300;
+	display: flex;
+	justify-content: space-around;
+}
+
+.container p {
+	color: grey;
+}
+
+h1 {
+	margin-top: 50px;
+	margin-left: 170px;
+	margin-bottom: 50px;
+}
+
+.col-1 {
+	width: 60%;
+	margin-left: 50px;
+	display: flex;
+	flex-flow: row wrap;
+	justify-content: space-between;
+	align-items: center;
+	text-align: center;
+	margin-bottom: 150px;
+}
+
+.sel {
+	text-align: left;
+	padding-top: 10px;
+	padding-left: 60px;
+}
+
+.col-2 {
+	width: 40%;
+}
+
+.form {
+	padding: 10px 10px;
+	text-align: center;
+	width: 400px;
+	height: 470px;
+	margin: 100px auto;
+	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+input {
+	width: 64%;
+	padding: 10px;
+	margin: 10px 0px;
+}
+
+.sel {
+	margin: 0;
```

Comment:
can we write this in a single line? Properties like margin and padding can be mentioned in a single line for all four sides